### ISSUE TITLE
` (code) inline handling

### ISSUE THREAD

**Issue by Shizen** [arve0/markdown-it-attrs#50]
If an attribute specification occurs inside a `` ` `` delimeted inline chunk *at the end of a line* `markdown-it-attrs` will pick it up.

I.e. if I do something like this `something{.example}`

The result would be something like...

```
<p class="example">
"I.e. if I do something like this "
<code>something</code>
</p>
```

I believe this is a bug :).  It is possibly related to issue #13.

**Comment by arve0** [comment]
Thanks for reporting, that is definitely a bug.

**Comment by arve0** [comment]
Fixed in `v1.2.1`.
